code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1521,7 +1521,7 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 		src << "<font color='red'><b>"+pick("The pain is excrutiating!", "Please, just end the pain!", "Your whole body is going numb!")
 
 	if(shock_stage >=60)
-		if(shock_stage == 60) emote("me",1,"'s body becomes limp.")
+		if(shock_stage == 60) custom_emote(1,"'s body becomes limp.")

```

Comment:
Actually, wait. If I'm not mistaken, won't this mess up custom_emotes given through "me [foo]"? I'll make this an apostrophe check, then, so that we don't mess those up.